page03_contact.php

```diff
@@ -2,7 +2,7 @@
 <html lang = "eng">
```

Comment:
code works; connect.php is connected and no longer has an error colour of red over each '/div' code